_plugins/sheafy.rb

```diff
@@ -25,12 +25,12 @@ def self.render_header(resource, level)
     HEADER
   end
 
-  RE_IMPORT_TAG = /{%\s*import (?<slug>.+?)\s*%}/
+  RE_INCLUDE_TAG = /@include{(?<slug>.+?)}/
```

Comment:
Should we allow only one per line? I.e. `/^@include{(?<slug>.+?)}$/`

Maybe that's premature to think about, but perhaps that will save us some heartache down the line when we try to render nodes as separate paragraphs... :thinking:

Let's just keep this in mind.